packages/ripple-binary-codec/src/enums/definitions.json

```diff
@@ -931,6 +948,36 @@
         "type": "UInt64"
       }
     ],
+    [
+      "XChainClaimID",
+      {
+        "nth": 20,
+        "isVLEncoded": false,
+        "isSerialized": true,
+        "isSigningField": true,
+        "type": "UInt64"
+      }
+    ],
+    [
+      "XChainAccountCreateCount",
+      {
+        "nth": 21,
+        "isVLEncoded": false,
+        "isSerialized": true,
+        "isSigningField": true,
+        "type": "UInt64"
+      }
+    ],
```

Comment:
Are there docs I can cross-reference for these fields?

Reply:
All the definitions.json changes were done with [xrpl-codec gen](https://github.com/RichardAH/xrpl-codec-gen), which reads the code and populates this file for you. So there's no code, but you can trust that it's accurate.

Reply:
Here's an explanation of what this file is supposed to look like: https://xrpl.org/serialization.html#definitions-file